src/eva/vision/data/datasets/segmentation/lits.py

```diff
@@ -49,6 +51,7 @@ def __init__(
         root: str,
         split: Literal["train", "val", "test"] | None = None,
         transforms: Callable | None = None,
+        fix_orientation: bool = True,
```

Comment:
maybe we should put it as a class state variable (like we do on the splits) because i think people wont want to change it

Reply:
done